astropy/utils/tests/test_console.py

```diff
@@ -155,6 +155,7 @@ def test_progress_bar_as_generator():
     assert sum == 1225
 
 
+@pytest.mark.slow
```

Comment:
test_progress_bar_func.func also sleeps (though not as long)

Reply:
For this one, `sleep` time is actually not dominant, so I think it should still be marked as slow with no other changes.